.gitignore

```diff
@@ -1,3 +1,4 @@
-src/*
```

Comment:
I'd prefer not to do it as I (_and probably many potential users_) prefer my solution for generating assets. It's a great initiative to offer functionality, which can help someone, but it should stay optional, and the "core Bender" should remain "tool agnostic". The example files below are not necessary, it all can be done by more detailed instructions in README.md

What's your opinion, Pavel @pavelhusa?